Code/Tools/FBuild/FBuildWorker/Worker/WorkerSettings.cpp

```diff
@@ -82,10 +82,8 @@ void WorkerSettings::Load()
         }
 
         // settings
-        uint32_t mode;
-        f.Read( mode );
-        m_Mode = (Mode)mode;
-        f.Read( m_NumCPUsToUse );
+        f.Read( m_NumCPUsToUseWhenIdle );
+		f.Read(m_NumCPUsToUseDedicated);
```

Comment:
This looks like a bug. We've set the versions such that we'll load the old config, but we aren't correctly handling the old format.